### ISSUE TITLE
Bad signature when using `Unpack` with empty `TypedDict`

### ISSUE THREAD

**Issue by Viicos** [microsoft/pyright#8712]
Really minor issue:

Code sample in [pyright playground](https://pyright-play.net/?reportMissingModuleSource=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoCqKCAhgMYDWANFACqICmAJgCJJkwBQ3ZANiQGcBUAKLZ4ACnoJmbDjRhgYJXgF4AYioEMAlAC5OUI1FJDuTBsCjAArijISAVI4oB3EiDQC9hYuQoA2mII8AC6%2BiaCAtyctvYSOpxAA)

```python
from typing import Unpack, TypedDict


class Empty(TypedDict, total=False):
    pass

def func(**kwargs: Unpack[Empty]): pass


func()
```

Produces:

![image](https://github.com/user-attachments/assets/3bd9fcb4-e529-429a-8387-867081dc13fb)

maybe `() -> None` could be more appropriate.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release of pyright.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.376.
